### ISSUE TITLE
A more up to date minetweaker script syntax document.

### ISSUE THREAD

**Issue by alperen010010** [jakimfett/Minechem#819]
The current one https://github.com/jakimfett/Minechem/blob/master/scripts/Minetweaker.zs was uploaded a year ago and it doesnt show some features like setting the color of molecules.

**Comment by way2muchnoise** [comment]
Feel free to update it and make a PR for it. I don't either @jakimfett or myself have time do that atm.

**Comment by alperen010010** [comment]
I cant

**Comment by ChristyKGrnKnght** [comment]
Even the outdated example minetweaker script syntax is non-available now! I'd love to be able to use this, but I'm late to the game. Can anyone give me a sample?

**Comment by way2muchnoise** [comment]
@ChristyKGrnKnght the example scripts are still therehttps://github.com/iopleke/Minechem/blob/master/scripts/Minetweaker.zs

**Comment by Blue64** [comment]
yeah, the Project Lead Admin changed, as a result, the old links broke due to the old Lead Admin's name being a part of the URL, just gotta use the correct name (found in the URL up top, or you can look at the top of the page to see who is the current Lead Admin of the Repo you are in:
```
iopleke/Minechem 
Lead Admin/Repo
```

**Comment by Blue64** [comment]
P. S. was the MT code ever updated on this repo?

**Comment by jakimfett** [comment]
Not to my knowledge. There haven't been any active pull requests in quite a while.
